redash/models/__init__.py

```diff
@@ -821,20 +821,42 @@ def evaluate(self):
     def subscribers(self):
         return User.query.join(AlertSubscription).filter(AlertSubscription.alert == self)
 
-    def render_template(self):
-        if not self.template:
+    def render_template(self, template=None):
+        if template is None:
             return ''
+
         data = json_loads(self.query_rel.latest_query_data.data)
-        context = {'rows': data['rows'], 'cols': data['columns'], 'state': self.state}
-        return mustache_render(self.template, context)
+        host = base_url(self.query_rel.org)
+
+        col_name = self.options['column']
+        if data['rows'] and col_name in data['rows'][0]:
+            result_value = data['rows'][0][col_name]
+        else:
+            result_value = ''
```

Comment:
```suggestion
            result_value = None
```

`None` feels like a better fallback value in such case.